tests/common/common_vec.h

```diff
@@ -334,7 +317,8 @@ bool check_vector_size_byte_size(sycl::vec<vecType, N> inputVec) {
   if (count != N || !noexcept(inputVec.size())) {
     return false;
   }
-  count = vector_swizzle_check<vecType, N>::get_swizzle(inputVec).size();
+  DO_OPERATION_ON_SWIZZLE(inputVec, count, .size();)
```

Comment:
I would suggest to put trailing `;` into the macro to avoid repeating it everywhere

Reply:
[1bd3720](https://github.com/KhronosGroup/SYCL-CTS/pull/801/commits/1bd372017ddb1be9c9876d19d7e13900dfdd11d9)

Reply:
While I understand the economy, on the other hand I find this confusing while not looking inside the macro.
It might also confuse the indentation done with editors and formatters.
But if it works with `clang-format` I am OK with this.